07-react-1/classworks/Emelyanov_Egor/src/pages/home/CommunityPosts.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import moment from "moment";
+
+import lukashenkoImg from './lukashenko.jpg';
```

Comment:
`lukashenko.jpg` here but `lukasenko.jpg` in another file

Reply:
while filename is 
![image](https://user-images.githubusercontent.com/23057064/89271699-bc645180-d645-11ea-9af7-80eaa5636a2f.png)